db/migrate/20160816150906_add_email_alert_signup_content_ids_to_topics.rb

```diff
@@ -0,0 +1,9 @@
+class AddEmailAlertSignupContentIdsToTopics < ActiveRecord::Migration
+  def change
+    add_column :topics, :email_alert_signup_content_id, :string
+
+    Topic.all.each do |topic|
```

Comment:
Also added an index with a unique constraint for that column. We don't appear to have one for content_id but it felt like it should have one…